features/step_definitions/basic_steps.rb

```diff
@@ -10,7 +10,7 @@
     fill_in(field, with: text)
 end
 
-Given("I fill in {string} with 1st of Jan, 2018") do |field|
+Given("I fill in {string} with Jan, 2018") do |field|
     element = page.find_by_id(field)
     element.send_keys :up
     element.send_keys :tab
```

Comment:
What exactly are we checking for here? If up and tab button works to change date?

Reply:
Filling in the form with the Month and Year. Simulating the key strokes to make that happen as in fill_in form didn't work for a dual field.